docs/CHANGELOG.rst

```diff
@@ -44,91 +44,97 @@ and anyone can contribute to it.
 It is possible to perform full-scale analyses and contribute to the production
 of CTA performances for different science cases or perform studies that require big amounts of data.
 
+
 Contributors
 ++++++++++++
 
-- Michele Peresano (`@HealthyPear`_)
+@HealthyPear and @kosack
 
 What is changed since v0.4.0.post1
 ++++++++++++++++++++++++++++++++++
 
 Pull-requests containing changes of multiple nature are repeated.
 
-🚀 General features
-^^^^^^^^^^^^^^^^^^^
+- fixed missing import of QTable (#188) @kosack
```

Comment:
these PRs do not seem to have been labeled: if they lack one of the labels defined in  the release drafter workflow they won't get classified in the updated draft